arduino/control-modules/i2c-reader.js

```diff
@@ -2,14 +2,16 @@ const eventBus = require('../../events/event-bus')
 const { I2C_DATA } = require('../sensor-keys')
 const { I2C_DATA_RECIEVED } = require('../../events/event-keys')
 
-function handlChannel (five, channelNumber) {
-  five.io.i2cRead(channelNumber, 27, function (ArrayOfBytes) {
-    var length = ArrayOfBytes.length
-    var message = ''
+function handlChannel (board, channelNumber) {
+  board.io.i2cRead(channelNumber, 27, function (arrayOfBytes) {
```

Comment:
it will read from the _slave_ channel. but in the https://github.com/spaceshipyard/arduino-sketches/blob/master/uno2-motor-sensor-bridge-i2c#L34 the firmware initied as the master
```
Wire.begin(); //without address argument it is the master.
```
in the same time johnny-five works as another master because of the following:
```
board.io.i2cConfig()
```